app/views/task-list.html

```diff
@@ -0,0 +1,11 @@
+{% extends "layout.html" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-l question-header">{{ i18n.taskList.header }}</h1>
```

Comment:
remove the `question-header` class here

Reply:
You sure? You put it there

Reply:
Only for the question header h1, this is a task list header

Reply:
Oh wait thought it was the other template

Reply:
Looks like a copy and paste error 😱